src/js/components/AppListStatusFilterComponent.jsx

```diff
@@ -101,25 +102,44 @@ var AppListStatusFilterComponent = React.createClass({
     }
   },
 
+  getStatusCountBadge: function (id, appStatus) {
+    var state = this.state;
+
+    if (!state.appsStatusesCount[appStatus]) {
+      return null;
+    }
+
+    return (
+      <label htmlFor={id} className="label visible">
+        {state.appsStatusesCount[appStatus]}
```

Comment:
Let's `toLocaleString()` this guy as per https://github.com/mesosphere/marathon/issues/2650#issuecomment-157455695